src/internal/svg/coinbasePaySvg.tsx

```diff
@@ -12,7 +14,7 @@ export const coinbasePaySvg = (
       fillRule="evenodd"
       clipRule="evenodd"
       d="M10.0145 14.1666C7.82346 14.1666 6.04878 12.302 6.04878 9.99996C6.04878 7.69788 7.82346 5.83329 10.0145 5.83329C11.9776 5.83329 13.6069 7.33677 13.9208 9.30552H17.9163C17.5793 5.02774 14.172 1.66663 10.0145 1.66663C5.63568 1.66663 2.08301 5.39926 2.08301 9.99996C2.08301 14.6007 5.63568 18.3333 10.0145 18.3333C14.172 18.3333 17.5793 14.9722 17.9163 10.6944H13.9208C13.6069 12.6632 11.9776 14.1666 10.0145 14.1666Z"
-      fill="#f9fafb"
+      className={className}
```

Comment:
Doesn't this break default styling in things like `CheckoutButton`?

Reply:
I have set color `icon.foreground` by default

Reply:
This will break the default styling for this SVG when themes are applied. If you need a CoinbasePaySVG to update then we might have to create a new SVG for that